Rasa_Bot/domain.yml

```
@@ -125,6 +142,94 @@ responses:
+             We lopen samen door de volgende stappen:\n
+             Eerst denk je na over jezelf op dit moment.
+             Dan denk je na over jezelf over een paar maanden.
+             Daarna stel ik jou een aantal vragen hierover.\n
+             Geen stress, ik loop er stap voor stap met je doorheen.
+            "
+  utter_current_smoker:
+    - text: "Eerst kijken wij wat voor roker jij op dit moment bent.
+             Ik laat je zo een lijst met woorden zien.
+             Welke woorden passen bij jou als roker?
+             Roken is...
+            "
+  utter_current_mover:
+    - text: "We gaan nu dezelfde oefening doen voor bewegen.
+             Welke woorden vind jij passen bij bewegen?
+             Bewegen is ...
+            "
+  utter_smoking_word_list:
+    - text: "Fijn - lekker - eng - past bij mij- plezierig - belonend - stressvol
+             - moet ik doen - helpt mij - bedreigend - helpt mij ontspannen - vermoeiend
+             - cool - iets wat ik wil blijven doen - onderdeel van mijn leven - vies
+             - prettig - gevaarlijk - verslavend - belangrijk - niet belangrijk
+             - hoort niet bij mij - stom - stoer - gezellig - duur - sexy
+             - ongezond - niet zo heel erg
+            "
+  utter_moving_word_list:
+    - text: "Fijn - lekker - past bij mij - plezierig - belonend - geen zin in
+             - helpt mij ontspannen - vermoeiend - cool
+             - onderdeel van mijn leven - prettig - belangrijk - niet belangrijk
+             - hoort niet bij mij - stom - goed - gezellig - kost tijd
+             - stressvol - helpt mij - iets wat ik meer wil doen
+             - vind ik niet leuk - vind ik leuk - doet pijn - gezond
+             - moet voor mijn gezondheid - makkelijk - doe ik veel - straf
+             - gaat niet - lastig - houdt mij jong - eng"
+  utter_ask_pick_smoker_words_form_picked_words:
+    - text: "Kies 3 tot 5 woorden uit de lijst en typ ze in de chat."
+  utter_ask_pick_mover_words_form_picked_words:
+    - text: "Kies 3 tot 5 woorden uit de lijst en typ ze in de chat:"
+  utter_picked_smoker_words:
+    - text: "Deze woorden passen bij jou als roker: {picked_words}, klopt dat?"
+  utter_picked_mover_words:
+    - text: "{picked_words} heb jij gekozen, klopt dat?"
+  utter_ask_why_picked_smoker_words_form_why_picked_words:
+    - text: "Mooi. Waarom heb je deze woorden gekozen voor roken?"
+  utter_ask_why_picked_mover_words_form_why_picked_words:
+    - text: "Duidelijk. Waarom heb je deze woorden gekozen voor bewegen?"
+  utter_why_picked_words_response:
+    - text: "Goed dat je hier zo over na kunt denken."
+
+  utter_self_view_intro:
+    - text: "Nu krijg je twee zinnen te zien.
+             Vul ze aan met wat het meest bij jou past."
+  utter_see_myself_as_smoker_options:
+    - text: "Ik zie mijzelf als...
+             een roker - een niet-roker - iemand die stopt met roken"
```

```suggestion
             een roker (1) - een niet-roker (2) - iemand die stopt met roken (3)"
```
We enforce only 1 answer. And question becomes: ' Kies de optie die het meeste bij je past en typ het nummer in de chat'.

---

We don't need to store the numbers, but makes it easier to type the response. And also check if the user responded correctly.

---

This will be done in: https://github.com/PerfectFit-project/virtual-coach-issues/issues/178